app/controllers/cart_controller.rb

```diff
@@ -30,7 +30,7 @@ def destroy
     @cart.delete_accessory(@accessory)
     session[:cart] = @cart.contents
 
-    flash[:success] = "Successfully removed #{@accessory.title} from your cart."
+    flash[:success] = "Successfully removed #{view_context.link_to(@accessory.title, @accessory)} from your cart.".html_safe
```

Comment:
What an interesting way to get the link into the flash. Thanks for figuring this out.